app/controllers/admin/base_controller.rb

```diff
@@ -58,7 +58,11 @@ def fetch_counts
       @admins = User.agents #can_receive_ticket.tagged_with(current_user.team_list, :any => true)
     else
       topics = Topic.all
-      @admins = User.agents
+      first = [current_user]
+      @admins = [*first, *(User.agents - first)]
+      #@admins = User.agents
+      #first = [current_user]
+      #puts [*first, *(@admins.sort - first)].inspect
```

Comment:
Can you remove this commented out piece of code?

Also, wouldn't 
```
# move currently signed in user to front of list
@admins = first + (User.agents - first)
```

or something similar, be a bit simpler? (The splat operator can be rather confusing to people...)